<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailedMenu</class>
 <widget class="QWidget" name="DetailedMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>果粒橙外卖</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Image.qrc">
    <normaloff>:/image/lt_cml.jpg</normaloff>:/image/lt_cml.jpg</iconset>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;菜品名称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff; /* 默认背景色，使用 #fad37f */
    border: 2px solid rgba(99, 132, 233, 0.8); /* 边框颜色 */
    border-radius: 9px; /* 圆角 */
    color: #000000; /* 文本颜色，黑色 */
    font-size: 9px;
}

QPushButton:hover {
	color: #ffffff;
    background-color: rgba(99, 132, 233, 0.8); /* 悬浮时背景颜色 */
    border: none; /* 悬浮时无边框 */
}

QPushButton:pressed {
    background-color: #b0c4de; /* 按下时背景颜色，稍微偏亮的蓝色 */
    border-color: #4682b4; /* 按下时边框颜色 */
    transform: scale(0.95); /* 按下时按钮略微缩小 */
}

</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>720</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff; /* 默认背景色，白色 */
    border: 2px solid rgba(99, 132, 233, 0.8); /* 边框颜色 */
    border-radius: 15px; /* 圆角 */
    color: #081439; /* 文本颜色，深蓝色 */
    font-size: 11px;
}

QPushButton:hover {
	color: #ffffff;
    background-color: rgba(99, 132, 233, 0.8); 
    border: none; /* 悬浮时无边框 */
}

QPushButton:pressed {
    background-color: #b0c4de; /* 按下时背景颜色，稍微偏亮的蓝色 */
    border-color: #4682b4; /* 按下时边框颜色 */
    transform: scale(0.95); /* 按下时按钮略微缩小 */
}
</string>
   </property>
   <property name="text">
    <string>加入购物车</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;总计价格：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>314</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>两人份</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;菜品规格：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;口味&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>不辣</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>一人份</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2" colspan="3">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;分量&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>微辣</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string>特辣</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>481</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;这道菜色泽鲜艳，香气扑鼻，口感细腻，令人垂涎欲滴，回味无穷。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/bg_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>670</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_4</zorder>
  <zorder>spinBox</zorder>
 </widget>
 <resources>
  <include location="../Image.qrc"/>
 </resources>
 <connections/>
</ui>
